README.md

```diff
@@ -1,23 +1,52 @@
-# Hanami
+[![Gem Version](https://badge.fury.io/rb/hanami.svg)](http://badge.fury.io/rb/hanami)
+[![Build Status](https://secure.travis-ci.org/hanami/hanami.svg?branch=master)](http://travis-ci.org/hanami/hanami?branch=master)
+[![Coverage](https://coveralls.io/repos/hanami/hanami/badge.svg?branch=master)](https://coveralls.io/r/hanami/hanami)
+[![Code Climate](https://codeclimate.com/github/hanami/hanami.svg)](https://codeclimate.com/github/hanami/hanami)
+[![Dependencies](https://gemnasium.com/hanami/hanami.svg)](https://gemnasium.com/hanami/hanami)
+[![Inline docs](http://inch-ci.org/github/hanami/hanami.svg)](http://inch-ci.org/github/hanami/hanami)
+
+# Hanami :cherry_blossom:
 
 The web, with simplicity.
 
 ## Frameworks
 
-Hanami combines small yet powerful frameworks:
+Hanami is a **full-stack** Ruby web framework.
+It's made up of smaller, single-purpose libraries.
 
-* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
-* [**Hanami::Router**](https://github.com/hanami/router) - Rack compatible HTTP router for Ruby
+This repository is for the full-stack framework,
+which provides the glue that ties all the parts together.
+
+Here are the smaller single-purpose frameworks:
+
+* [**Hanami::Model**](https://github.com/hanami/model) - Persistence with entities, repositories and data mapper
+* [**Hanami::View**](https://github.com/hanami/view) - Presentation with a separation between views and templates
+* [**Hanami::Controller**](https://github.com/hanami/controller) - Full featured, fast and testable actions for Rack
 * [**Hanami::Validations**](https://github.com/hanami/validations) - Validations mixin for Ruby objects
+* [**Hanami::Router**](https://github.com/hanami/router) - Rack compatible HTTP router for Ruby
 * [**Hanami::Helpers**](https://github.com/hanami/helpers) - View helpers for Ruby applications
 * [**Hanami::Mailer**](https://github.com/hanami/mailer) - Mail for Ruby applications
-* [**Hanami::Model**](https://github.com/hanami/model) - Persistence with entities, repositories and data mapper
 * [**Hanami::Assets**](https://github.com/hanami/assets) - Assets management for Ruby
-* [**Hanami::View**](https://github.com/hanami/view) - Presentation with a separation between views and templates
-* [**Hanami::Controller**](https://github.com/hanami/controller) - Full featured, fast and testable actions for Rack
+* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
 
 These components are designed to be used independently or together in a Hanami application.
-If you aren't familiar with them, please take time to go through their READMEs.
+
+## Installation
+__Hanami__ supports Ruby (MRI) 2.3+
```

Comment:
@Neodelf We don't support JRuby for `hanami` yet. All the other `hanami-*` libraries already support it.